### PR TITLE
this might have shaved half the time off my build

### PR DESCRIPTION
**Pull request by jschneidereit**

### CODE REVIEW

File: gradle.properties [hunk 1]
```diff
@@ -10,7 +10,7 @@ gradle.publish.key=xx
 gradle.publish.secret=xx
 
 # See https://dev.to/jmfayard/configuring-gradle-with-gradle-properties-211k
-org.gradle.parallel=false
+org.gradle.parallel=true
```
Comment (jschneidereit) [inline gradle.properties:13]: https://scans.gradle.com/s/pywo4r5glfll2/tests/failed

add a override to kotest-extensions:kotest-extensions-http

**Comment by sksamuel** [comment]
parallel should stay off in the release builds, so we need to make sure
it's overriden there.

**Comment by sksamuel** [comment]
Will merge once out of draft and override is in the git workflow

**Comment by sksamuel** [comment]
@jschneidereit do you want me to add the override ?

**Comment by jschneidereit** [comment]
Oh yeah sure, or I can do it in a few hours 👍

**Comment by sksamuel** [comment]
Look like release already has parallel overrides. Can merge.

**Pull request merged**